submissions/Dovahkiin1991/oop-exercise/index.js

```diff
@@ -0,0 +1,76 @@
+import { print } from './js/lib.js';
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Dovahkiin1991/a-tiny-JS-world
+   Web app: https://dovahkiin1991.github.io/a-tiny-JS-world/
+   */
+// ======== OUTPUT ========
+class Creature {
+    constructor(species, name, gender, legs, hands, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.legs = legs;
+        this.hands = hands;
```

Comment:
Hands are inherited by all animals. They just do not need them.
OOP helps to define properties exactly where they needed. OOP in general and class syntax in particular is more powerful thing than just a tool to define default values for numerous properties.